pkg/plugins/plugins.go

```diff
@@ -108,6 +108,7 @@ type JSONData struct {
 	ID           string       `json:"id"`
 	Type         Type         `json:"type"`
 	Name         string       `json:"name"`
+	Alias        string       `json:"alias,omitempty"`
```

Comment:
since hardcoding is the hack... why move this to another level that will make it harder in the future?

IIUC, to remove the hardcoding hack, we should just need to add it to the cue schema

Reply:
For internal/hardcoded plugins sure -- it makes no difference.  But if we want this to be driven by the plugin configuration for external plugins eg (dlopes7-appdynamics-datasource, etc) where would it get set?

I honestly do not feel very strongly about any particular solution -- there are many many ways to skin this chicken -- just do not want to keep kicking the can down the road and never finding any approach.

Reply:
> believe it's part of plugin.json

Fundamentally I think it should be, having it hardcoded for phlare is just a strategy to gain some confidence right?  We still need a solution that allows future plugins to rebrand without having a clear path forward.

Reply:
100% agree - I'm just saying that we should have things in their rightful place _for now_ and when we have the confidence as you say, then we go all in. `JSONData` is our model for plugin.json and since we're hardcoding things for now, custom fields like that get put on the top level